modules/images/dominant-color/load.php

```diff
@@ -117,7 +117,7 @@ function dominant_color_img_tag_add_dominant_color( $filtered_image, $context, $
 
 	if ( ! empty( $image_meta['dominant_color'] ) ) {
 		$data .= sprintf( 'data-dominant-color="%s" ', esc_attr( $image_meta['dominant_color'] ) );
-		$style = 'style="--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';" ';
+		$style = '--dominant-color: #' . esc_attr( $image_meta['dominant_color'] ) . ';" ';
```

Comment:
- Line 131 appears to add `$style` just after `style="`. Since `$style` ends with `;" `, this will close the `style` value, breaking other inline styles.
- This also produces extra spacing (presumably unintended) between `data-dominant-color` and `style`. Not a big deal, but might be unintended.

See the results of [this 3v4l](https://3v4l.org/ZX70X).

Reply:
Thanks for catching this
Fixed and remove extra space